components/ProjectSearch.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { FontAwesome } from '@expo/vector-icons';
+import styled from 'styled-components/native';
+
+
+const ProjectSearch = () => {
+
+    const SearchView = styled.View`
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    margin-right: 54px;
+    `
```

Comment:
styled components should be outside of the component definition as they need to be static